web/src/elements/forms/FormGroup.ts

```diff
@@ -8,11 +8,26 @@ import PFForm from "@patternfly/patternfly/components/Form/form.css";
 import PFFormControl from "@patternfly/patternfly/components/FormControl/form-control.css";
 import PFBase from "@patternfly/patternfly/patternfly-base.css";
 
+/**
+ * Form Group
+ *
+ * Mostly visual effects, with a single interaction for opening/closing the view.
+ *
+ */
+
+/**
+ * TODO: Listen for custom events from its children about 'invalidation' events, and
+ * trigger the `expanded` property as needed.
+ */
+
 @customElement("ak-form-group")
 export class FormGroup extends AKElement {
-    @property({ type: Boolean })
+    @property({ type: Boolean, reflect: true })
     expanded = false;
 
```

Comment:
Providing a means to modify the aria-label, rather than stick with the just the word  "Details." Specializing this field will both help users of screen readers in the future, and will  allow test suites to find specific form groups now.